lib/wam.coffee

```diff
@@ -9,6 +9,7 @@ os = require 'os'
 url = require 'url'
 unzip = require 'extract-zip'
 async = require 'async'
+cs = require 'cloudscraper'
```

Comment:
Since `cloudscraper` can replace `request` and all usages of `request` have been replaced by `cloudscraper`, there's no need to import `request` on line 3, right?

Reply:
Yes, there's no need to import request anymore, and I can remove it.